jupyterlab_pullrequests/__init__.py

```diff
@@ -19,6 +19,12 @@ def _load_jupyter_server_extension(server_app):
     from .base import PRConfig
     from .handlers import setup_handlers
 
+    log = server_app.log
     config = PRConfig(config=server_app.config)
-    setup_handlers(server_app.web_app, config)
+    setup_handlers(server_app, config)
     server_app.log.info("Registered jupyterlab_pullrequests extension")
+
+
+# for legacy launching with notebok (e.g. Binder)
+_jupyter_server_extension_paths = _jupyter_server_extension_points
+load_jupyter_server_extension = _load_jupyter_server_extension
```

Comment:
good start, there's also the json files that need to be deployed? or maybe we already do that...

Reply:
The JSON files were already well set. But this backport was missing